### ISSUE TITLE
[hdhr_parent].channel_map is not setting the map.

### ISSUE THREAD

**Issue by CraziFuzzy** [enternoescape/opendct#353]
I am unfamiliar with the way the options are supposed to work, but on the HDHomeRunDiscoveredDeviceParent there is an option to force the channelmaps on the device to a certain type on discovery.  This is not actually doing anything.  If I change the initial value in HDHomeRunDiscoveredDeviceParent.java to a value ("us-bcast", for instance), then it does set it to that - but it no matter what I set the property to in opendct.properties, it never returns anything but "".

**Comment by enternoescape** [comment]
Are you editing something like this? sagetv.device.parent.-503834129.channel_map=

It is left blank on purpose by default because if the value is set it will change the value on the device.

**Comment by enternoescape** [comment]
The code that uses that value is in opendct/capture/HDHRNativeCaptureDevice.java on line 114.

**Comment by CraziFuzzy** [comment]
Yes, I found where it was setup and such.  I was working in that file (and others) to enable the missing channel maps.  The channel_map property is not written to the .properties file, even blank - and adding it and setting it to absolutely anything also results in nothing happening (in fact, it's not even detecting invalid entries).  The only time I got that particular StringDeviceOption to return anything was when i changed it's 'value' in HDHomeRunDiscoveredDeviceParent.java to a value other than "", then it returned that value.

**Comment by enternoescape** [comment]
You're right, it's missing something important. I wonder when I broke that; I remember it working a little while ago. Let me fix that.
